.circleci/config.yml

```diff
@@ -38,11 +38,27 @@ jobs:
           name: Run tests
           command: yarn test:sdk
 
+  audit:
+    docker:
+      - image: cimg/node:16.14.2-browsers
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-yarn-{{ checksum "yarn.lock" }}
+            - v1-yarn-  
+      - run:
+          name: Run audit
+          command: |
+            yarn install --cache-folder ~/.cache/yarn
+            yarn audit:ci
```

Comment:
I think we can reuse the CircleCI workspace, same as how we did for the test job. This should make it faster:

```
  audit:
    docker:
      - image: cimg/node:16.14.2-browsers
    steps:
      - attach_workspace:
          at: .
      - run:
          name: Run audit
          command: yarn audit:ci
```